src/components/Main/Results/AgeBarChart.tsx

```diff
@@ -11,6 +11,7 @@ import { SeverityTableRow } from '../Scenario/SeverityTable'
 import { colors } from './DeterministicLinePlot'
```

Comment:
It is basically a complicated way to cast return value to a more narrow type.
But what if  someone on the other end of the translation pipe goes crazy and `t()` will actually return an object or an array? Everything will blow up.

I think this should be a runtime check with a throw. Not elegant, but safe.
Feel free to either one-off inline or to wrap into a function (`safeT()`? `tString()`? idk) with the correct return type, both compile-time and run-time.